docs/01-get-started/your-first-test.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Your first test 🧪
+description:
+  In Front-Commerce v3.4, we introduced a new way to test your application using
+  vitest. This guide will help you to create your first test to ensure your
+  application is working as expected.
+sidebar_position: 5
+---
+
+<p>{frontMatter.description}</p>
+
+<SinceVersion tag="3.4" />
+
+# Resources
+
+- [Vitest](https://vitest.dev/)
+- [React Testing Library](https://testing-library.com/docs/react-testing-library/intro/).
+- [@front-commerce/core/testing](/docs/3.x/api-reference/front-commerce-core/testing#createfrontcommerceproviderstub)
+- [@remix-run/testing](https://remix.run/docs/en/main/other-api/testing)
+
+:::important
```

Comment:
**thought:** It might be worth to note the globby patterns we're using to match test files somewhere as well, so that the integrator know the possibilities.